javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/OpenTelemetryInstaller.java

```diff
@@ -34,11 +37,19 @@ public void beforeAgent(Config config) {
   @SuppressWarnings("unused")
   public static synchronized void installAgentTracer(Config config) {
     if (config.getBooleanProperty(JAVAAGENT_ENABLED_CONFIG, true)) {
+
+      System.setProperty("io.opentelemetry.context.contextStorageProvider", "default");
+
       copySystemProperties(config);
```

Comment:
Shouldn't that be moved to the `!= noop` branch? There's no need to copy SDK config if you don't use the SDK at all

Reply:
yep, it's only useful there - thanks!